apps/desktop/desktop/src/main/java/bisq/desktop/main/content/mu_sig/create_offer/payment/MuSigCreateOfferPaymentController.java

```diff
@@ -198,7 +198,11 @@ void onTogglePaymentMethod(PaymentMethod<?> paymentMethod, PaymentMethodChipButt
         }
     }
 
-    public void onSelectAccount(Account<? extends PaymentMethod<?>, ?> account, PaymentMethod<?> paymentMethod) {
+    public void selectAccount(Account<? extends PaymentMethod<?>, ?> account, PaymentMethod<?> paymentMethod) {
+        doSelectAccount(account, paymentMethod);
+    }
+
+    void doSelectAccount(Account<? extends PaymentMethod<?>, ?> account, PaymentMethod<?> paymentMethod) {
```

Comment:
This method should be private. And the `void onSelectAccount` should be used form the View and calls the `doSelectAccount`. Idea is to have 2 clear interfaces: one for the internal view and one for external client. to avoid code duplication we use one private method for both.